sql/codegen_alps.go

```diff
@@ -89,20 +100,28 @@ func newALPSTrainFiller(pr *extendedSelect, db *DB) (*alpsFiller, error) {
 
 	var odpsConfig = &gomaxcompute.Config{}
 	var columnInfo map[string]*columnSpec
-	var meta metadata
 
 	// TODO(joyyoj) read feature mapping table's name from table attributes.
 	// TODO(joyyoj) pr may contains partition.
 	fmap := featureMap{pr.tables[0] + "_feature_map", ""}
+	var meta metadata
+	selectFields := pr.standardSelect.fields
```

Comment:
What if `SELECT *`?

Reply:
I know this problem. I try use 'desc table' clause to get fields but maxcompute report "not support instance type". I add a TODO here.

Reply:
Use `select * from table limit 1 `clause to get fields. if table is not empty, it works

Reply:
Cool~